pol-core/pol/bowsalut.cpp

```diff
@@ -186,62 +187,65 @@ void send_action_to_inrange( const Mobile::Character* obj, UACTION action,
 {
   bool build = false;
   Network::MobileAnimationMsg msg( obj->serial_ext );
-  WorldIterator<OnlinePlayerFilter>::InVisualRange( obj, [&]( Mobile::Character* zonechr ) {
-    if ( !build )  // only build if client in range
-    {
-      MobileTranslate::OldAnimDef oldanim;
-      oldanim.valid = true;
-      oldanim.action = static_cast<u16>( action );
-      oldanim.framecount = framecount;
-      oldanim.repeatcount = repeatcount;
-      oldanim.backward = static_cast<u8>( backward );
-      oldanim.repeatflag = static_cast<u8>( repeatflag );
-      oldanim.delay = delay;
-      MobileTranslate::NewAnimDef newanim;
-      bool supports_mount = true;
-      build = true;
-      MobileTranslate const* translate = nullptr;
-      for ( const auto& translates : Core::gamestate.animation_translates )
+  WorldIterator<OnlinePlayerFilter>::InRange(
+      obj, obj->update_range(),
```

Comment:
maybe a dumb question: is the update_range() here how much the client wants to see (like the client setting) or how big is their "effect" range?

Reply:
And bug found.. thx :)
obj->update_range() is the visibility distance so for chars their line of sight and for objects it could be higher due to multi footprint